package.json

```diff
@@ -16,6 +16,17 @@
       "inventory": "10.0"
     },
     "permissionSets": [
+      {
+        "permissionName": "records-editor.all",
+        "displayName": "quickMARC: view, edit",
+        "subPermissions": [
+          "records-editor.records.item.get",
+          "records-editor.records.item.put",
+          "ui-inventory.instance.view",
```

Comment:
why do you need `ui-inventory` perms? we need just BE permission to read instance item